test/support/generator/get-geospatial-addresses-generator.ts

```diff
@@ -0,0 +1,230 @@
+            CLASSIFICATION_CODE: 'test',
+            CLASSIFICATION_CODE_DESCRIPTION: 'test',
+            LOCAL_CUSTODIAN_CODE: 12345,
+            LOCAL_CUSTODIAN_CODE_DESCRIPTION: 'test',
+            COUNTRY_CODE: v.COUNTRY_CODE,
+            COUNTRY_CODE_DESCRIPTION: 'test',
+            POSTAL_ADDRESS_CODE: 'test',
+            POSTAL_ADDRESS_CODE_DESCRIPTION: 'test',
+            BLPU_STATE_CODE: 'test',
+            BLPU_STATE_CODE_DESCRIPTION: 'test',
+            TOPOGRAPHY_LAYER_TOID: 'test',
+            LAST_UPDATE_DATE: 'test',
+            ENTRY_DATE: 'test',
+            BLPU_STATE_DATE: 'test',
+            LANGUAGE: 'test',
+            MATCH: 12345,
+            MATCH_DESCRIPTION: 'test',
+            DELIVERY_POINT_SUFFIX: 'test',
+          },
+        },
+      ],
+    }));
+
+    const getAddressOrdnanceSurveyMultipleResponse: GetAddressOrdnanceSurveyResponse = {
+      header: {
+        uri: 'test',
+        query: 'test',
+        offset: 0,
+        totalresults: 0,
+        format: 'test',
+        dataset: 'test',
+        lr: 'test',
+        maxresults: 100,
+        epoch: 'test',
+        lastupdate: 'test',
+        output_srs: 'test',
+      },
+      results: values.map((v) => ({
+        DPA: {
+          UPRN: 'test',
+          UDPRN: 'test',
+          ADDRESS: 'test',
+          BUILDING_NAME: v.BUILDING_NAME,
+          BUILDING_NUMBER: v.BUILDING_NUMBER,
+          ORGANISATION_NAME: v.ORGANISATION_NAME,
+          DEPENDENT_LOCALITY: v.DEPENDENT_LOCALITY,
+          THOROUGHFARE_NAME: v.THOROUGHFARE_NAME,
+          POST_TOWN: v.POST_TOWN,
+          POSTCODE: v.POSTCODE,
+          RPC: 'test',
+          X_COORDINATE: 12345,
+          Y_COORDINATE: 12345,
+          STATUS: 'test',
+          LOGICAL_STATUS_CODE: 'test',
+          CLASSIFICATION_CODE: 'test',
+          CLASSIFICATION_CODE_DESCRIPTION: 'test',
+          LOCAL_CUSTODIAN_CODE: 12345,
+          LOCAL_CUSTODIAN_CODE_DESCRIPTION: 'test',
+          COUNTRY_CODE: v.COUNTRY_CODE,
+          COUNTRY_CODE_DESCRIPTION: 'test',
+          POSTAL_ADDRESS_CODE: 'test',
+          POSTAL_ADDRESS_CODE_DESCRIPTION: 'test',
+          BLPU_STATE_CODE: 'test',
+          BLPU_STATE_CODE_DESCRIPTION: 'test',
+          TOPOGRAPHY_LAYER_TOID: 'test',
+          LAST_UPDATE_DATE: 'test',
+          ENTRY_DATE: 'test',
+          BLPU_STATE_DATE: 'test',
+          LANGUAGE: 'test',
+          MATCH: 12345,
+          MATCH_DESCRIPTION: 'test',
+          DELIVERY_POINT_SUFFIX: 'test',
+        },
+      })),
+    };
+
+    const getAddressOrdnanceSurveyEmptyResponse: GetAddressOrdnanceSurveyResponse[] = values.map(() => ({
+      header: {
+        uri: 'test',
+        query: 'test',
+        offset: 0,
+        totalresults: 0,
+        format: 'test',
+        dataset: 'test',
+        lr: 'test',
+        maxresults: 100,
+        epoch: 'test',
+        lastupdate: 'test',
+        output_srs: 'test',
+      },
+    }));
```

Comment:
Does this need to be an array?

Reply:
Good idea, no need for it to be an array.